classes/class-wc-connect-help-provider.php

```diff
@@ -687,7 +689,7 @@ public function page() {
 				<h2>
 					<?php _e( 'WooCommerce Connect Status', 'woocommerce' ); ?>
 				</h2>
-				<div id="wc-connect-admin-container"></div>
+				<div class="wc-connect-container" id="<?= $root_view ?>"></div>
```

Comment:
We can't use `<?=` since that requires PHP 5.4 and WordPress can run as low as 5.2.4.

Also, please esc_attr or esc_attr_e $root_view